packages/EasyBankFiles/src/Parsers/Bpay/Batch/Parser.php

```diff
@@ -84,7 +84,7 @@ public function getTransactions(): array
      */
     protected function processLine(int $lineNumber, string $line): void
     {
-        $code = $line[0] ?? '0';
+        $code = $line[0];
```

Comment:
We didn't need this since we skipping all empty lines in the `AbstractLineByLineParser::process()`
```
foreach ($contents as $line) {
            $line = \trim($line);

            if ($line === '') {
                continue;
            }

            $this->processLine($lineNumber, $line);
            $lineNumber++;
        }
```